mmv1/templates/terraform/resource.erb

```diff
@@ -38,6 +38,7 @@ import (
     "github.com/hashicorp/terraform-plugin-sdk/v2/diag"
     "github.com/hashicorp/terraform-plugin-sdk/v2/helper/logging"
 
+    "<%= import_path() -%>/services/compute"
```

Comment:
Got it- so we need this because `ComputeOperationError` lives in the new package. I wonder if we could transition that type to `tpgresource.CommonOpError` (not sure if they are identical)

Reply:
(definitely not needed here, just a consideration out-of-band)

Reply:
I will check. Thanks.

Reply:
This import for compute package is added here temporarily and will not be needed later once the Mmv1 generated compute resources also move to the compute package.